.travis.yml

```
@@ -3,11 +3,14 @@ services:
 - docker
 cache:
 - bundler
+before_install:
+- bash -c 'if [ command -v bundler ]; then bundle update --bundler fi'
```

```suggestion
- bash -c 'if [ command -v bundler ]; then bundle update --bundler; fi'
```

@majormoses - Pretty sure you're just missing a semicolon here.